pkg/sdk/streams_dto_builders_gen.go

```diff
@@ -0,0 +1,319 @@
+// Code generated by dto builder generator; DO NOT EDIT.
+
+package sdk
+
+import ()
+
+func NewCreateOnTableStreamRequest(
```

Comment:
It's generated, it behaves badly, because normally you would just have NewCreateXRequest, but because there are 4 variants here, e.g. CreateOnTable, it's concatenated like this. Because this is generated I wouldn't rename it; I would add it as known issue to our generator instead, but it's up to @sfc-gh-jcieslak .

Reply:
I'll fix the naming for Create constructors and remove import (), but I think we'll come back sooner or later to improving DTO generation where we'll go through all of the DTOs anyway